test/Frontend/Desugar.hs

```diff
@@ -1,20 +1,12 @@
 module Frontend.Desugar where
 
-import Data.Attoparsec.ByteString (parseOnly)
+import qualified Data.Attoparsec.ByteString as Parsec
+import qualified Juvix.Core.Common.NameSymbol as NameSym
 import qualified Juvix.Frontend.Parser as Parser
-import Juvix.Frontend.Types.Base
-import Juvix.FrontendDesugar (desugar)
+import qualified Juvix.FrontendDesugar as Desugar
+import qualified Juvix.FrontendDesugar.RemoveDo.Types as AST
```

Comment:
```suggestion
```
already import qualified as Desugared in the next line

Reply:
Thanks I removed the Desugared version